sdk/src/activity_context.rs

```diff
@@ -0,0 +1,212 @@
+use std::collections::HashMap;
+use std::sync::Arc;
+use std::time::{Duration as StdDuration, SystemTime};
+
+use prost_types::{Duration, Timestamp};
+use temporal_sdk_core_api::Worker;
+use temporal_sdk_core_protos::coresdk::{
+    activity_task,
+    common::{Payload, RetryPolicy, WorkflowExecution},
+    ActivityHeartbeat,
+};
+use tokio_util::sync::CancellationToken;
+
+/// Used within activities to get info, heartbeat management etc.
+#[derive(Clone)]
+#[allow(dead_code)]
```

Comment:
What was the dead code here? I'd prefer to avoid the annotation here and just dropped whatever was unused, can always add it back.